code/modules/urist/items/clothes/uristclothes.dm

```diff
@@ -633,21 +633,21 @@ Update 26/07/2014 - All generic clothing goes under obj/item/clothing/under/uris
 	desc = "It's a long-sleeve beige shirt with a red sweater-vest and brown trousers."
 	icon_state = "springm"
 	item_state = "springm"
-	item_color = "springm"
+	//item_color = "springm"
 
 /obj/item/clothing/under/urist/relaxedwearm
 	name = "pre-War male relaxedwear"
 	desc = "It's along-sleeve blue shirt with a greenish brown sweater-vest and slacks."
 	icon_state = "relaxedwearm"
 	item_state = "relaxedwearm"
-	item_color = "relaxedwearm"
+	//item_color = "relaxedwearm"
 
 /obj/item/clothing/under/urist/enclaveo
 	name = "Enclave officer uniform"
 	desc = "It's a standard Enclave officer uniform.<br>The outer layer is made of a sturdy material designed to withstand the harsh conditions of the wasteland."
 	icon_state = "enclaveo"
 	item_state = "enclaveo"
-	item_color = "enclaveo"
+	//item_color = "enclaveo"
```

Comment:
I realized just now all those item_colors were most likely entirely superfluous in the first place.